test/test-call-forwarding

```diff
@@ -12,7 +13,9 @@ def property_changed(property, value):
 def print_properties(cf):
 	properties = cf.GetProperties()
 
-	for p in properties:
+	for p in ["VoiceBusy", "VoiceNoReplyTimeout", "VoiceNotReachable",
+			"VoiceUnconditional", "VoiceNoReply",
+			"ForwardingFlagOnSim"]:
```

Comment:
I did this so that I could get rid of the first test case in:

https://wiki.ubuntu.com/Process/Merges/TestPlans/ofono/CallForwarding

This change now causes the script to verify that the interface and all the properties are properly exported.